scripts/quests/windurst/Blast_from_the_Past.lua

```diff
@@ -144,12 +144,13 @@ quest.sections =
     {
         check = function(player, status, vars)
             return status == xi.questStatus.QUEST_COMPLETED and
-                not xi.quest.getMustZone(player, xi.questLog.WINDURST, xi.quest.id.windurst.NOTHING_MATTERS)
+                xi.quest.getMustZone(player, xi.quest.log_id.WINDURST, xi.quest.id.windurst.NOTHING_MATTERS)
```

Comment:
This needs to be rebased with current code, the enum change here is deprecated

Reply:
Thanks! - Updated to `xi.questLog`

Presumably the examples in lua_baseentity.cpp should be updated? Or am I reading that wrong?